connectors/es/utils.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+
+
+def extract_index_or_alias(response, name):
+    index = response.get(name, None)
+
+    if index:
+        return index
```

Comment:
A bit more concise, since we don't reuse `index` anywhere
```suggestion
    if name in response:
        return response[name]
```